rpc/Cargo.toml

```
@@ -27,6 +27,7 @@ sp-blockchain = "2.0.0"
 sp-block-builder = "2.0.0"
 sp-consensus = "0.8.0"
 sp-consensus-babe = "0.8.0"
+sp-keystore = { git = "https://github.com/paritytech/substrate.git", rev = "77007ed87727a761169d4c57cbeae0151d9efb81" }
```

```suggestion
sp-keystore = "2.0.0"
```
it will be replaced by the patch of root cargo.toml, we don't need to specified the revision except the root cargo.toml